product_details/storage.py

```diff
@@ -173,14 +175,14 @@ def update(self, name, content, last_modified):
             log.debug('Writing new copy of %s to %s.' % (
                 name, self.json_dir))
             tf = tempfile.NamedTemporaryFile(delete=False)
-            tf.write(content)
+            tf.write(binary_type(content.encode('utf8')))
```

Comment:
Is `binary_type` necessary here? From what I just read, in Python 3 encode returns a `bytes` object already, and in Python 2 it returns an 8-bit `str`.